step-api/pom.xml

```diff
@@ -41,6 +41,11 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>structs</artifactId>
+            <version>1.0-20160312.085007-3</version>
```

Comment:
Before this merges, yes. This is considered a part of the exit criteria for the structs plugin.